facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java

```diff
@@ -357,4 +397,19 @@ public TimeoutConfiguration getTimeoutConfiguration() {
     public void setTimeoutConfiguration(TimeoutConfiguration timeoutConfiguration) {
         this.timeoutConfiguration = timeoutConfiguration;
     }
+
+    /**
+     * Shutdown thread pool executor for all facades.
+     *
+     */
+    public static void shutdown() {
```

Comment:
Where is that method call?

On Wed, May 18, 2016 at 11:07 AM, Marek <notifications@github.com> wrote:

> In
> facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java
> <https://github.com/lightblue-platform/lightblue-migrator/pull/350#discussion_r63721080>
> :
>
> > @@ -357,4 +397,19 @@ public TimeoutConfiguration getTimeoutConfiguration() {
> >      public void setTimeoutConfiguration(TimeoutConfiguration timeoutConfiguration) {
> >          this.timeoutConfiguration = timeoutConfiguration;
> >      }
> > +
> > +    /**
> > +     * Shutdown thread pool executor for all facades.
> > +     *
> > +     */
> > +    public static void shutdown() {
>
> True. But you still need a method to call. ServiceFacade.shutdown() is
> that method.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/350/files/17b676bee35728c0487e52415290b9e02d5b9cc0#r63721080>
>